cassandra-network-partition-test/main.star

```diff
@@ -40,7 +40,7 @@ def simulate_network_failure(plan, cassandra_run_output):
         command = ["/bin/sh", "-c", check_un_nodes],
     )
 
-    plan.wait(check_un_nodes_recipe, "output", "==", str(len(cassandra_run_output["node_names"])-1))
+    plan.wait(check_un_nodes_recipe, "output", "==", str(len(cassandra_run_output["node_names"])-1), timeout="5m")
```

Comment:
nit: could you use `named arguments` in all arguments here? that will help users to understand the code